### DISCUSSION TITLE
Table apperance

### DISCUSSION THREAD

**Discussion by thorinas** [Budibase/budibase#13707] [Help] [answered]
Hi, 

Im wondering if somone could help. 

im struggleing to understand how i can get my table to look like the ones in the templates please see below,

Mine
![image](https://github.com/Budibase/budibase/assets/71787922/d4624c39-db4d-4446-a935-dc3f252a52c7)

Templates 
![image](https://github.com/Budibase/budibase/assets/71787922/c4a702fc-3588-4fa2-9f13-0fac0c9afdfb)

Is this because Im using the builtin tables and not something like MySQL.

Thank you in advance

**Comment by mikesealey** [comment, marked as answer]
Hi @thorinas 

As Budibase grows, occasionally changes and improvements are made that aren't immediately reflected in the documentation and templates. Work will be taking place in the coming weeks to remedy this. 

In the meantime, you could look into using Custom CSS ([documentation here](https://docs.budibase.com/docs/custom-css)) to adjust the styling of your tables how you see fit. [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/CSS) and [W3Schools](https://www.w3schools.com/w3css/default.asp) both offer great resources for styling. You should also look into how to _Inspect Element_ in your chosen browser to help with writing custom styling.
